doc/Caliptra.md

```diff
@@ -986,83 +988,82 @@ Each of Caliptra's internal anti-rollback fuse banks shall support a minimum cou
+Mitigation against SCA is not trivial and may be implemented in a variety of ways. [[Reference 7](#ref-7)] provides a comprehensive overview of methods and techniques used in various SCA as well as recommendations for countermeasures against such attacks (including feasibility and applicability). Additionally, there are academic papers available from NIST and other resources that discuss SCA and their countermeasures.
 
-## Compliance and Certification Requirements
+## Compliance and certification requirements
 
-Due to the Identity service surface offered to other SoC subsystems, a Caliptra RTM may fall under the ToE (Target of Evaluation) of an application that wishes to attain a specific compliance level for business reasons.
+Due to the identity service surface offered to other SoC subsystems, a Caliptra RTM may fall under the Target of Evaluation (ToE) of an application that wishes to attain a specific compliance level for business reasons.
 
-It is important to highlight it’s not necessary for the RTM itself to unilaterally attain e.g. FIPS 140-3 L3. It is only relevant insofar the RTM is included in the “bag” that wants to obtain a compliance certification. For example, if a cloud provider wants to FIPS-certify PCIe link encryption in transit rooted to an ASIC identity emanating from a Caliptra RTM.
+It is important to highlight that it’s not necessary for the RTM itself to unilaterally attain (for example, FIPS 140-3 L3). It is only relevant when the RTM is included in the “bag” that wants to obtain a compliance certification. For example, it is relevant when a cloud provider wants to FIPS-certify PCIe link encryption in transit rooted to an ASIC identity emanating from a Caliptra RTM.
 
-Refer to [Reference 8](#ref-8) for requirements related to Keys, Entropy, and Random Bits and cryptographic modules and algorithms.
+See [Reference 8](#ref-8) for requirements related to keys, entropy, random bits, cryptographic modules, and algorithms.
 
-### Known Answer Test (KAT) Support
+### Known Answer Test (KAT) support
 
-In order to certify a cryptographic module, pre-operational self-tests must be performed when the system is booted. Implementation of KATs are required for FIPS certification. However, regardless of FIPS certification, it is considered a security best practice to ensure that the supported cryptographic algorithms are functioning properly so as to guarantee correct security posture.
+To certify a cryptographic module, pre-operational self tests must be performed when the system is booted. Implementing KATs is required for FIPS certification. However, regardless of FIPS certification, it is considered a security best practice to ensure that the supported cryptographic algorithms are functioning properly to guarantee correct security posture.
```

Comment:
Nit: I feel like "self-test" is more idiomatic than "self test", in the given context. Is also hyphenated in the below bullet list.

Reply:
updated